tensorflow/python/ops/nn_ops.py

```diff
@@ -4687,7 +4687,7 @@ def erosion2d_v2(value,
                  strides,
                  padding,
                  data_format,
-                 dilations,
```

Comment:
Changing parameter names at this point is not allowed as the TF2 API has to be backwards compatible.